recipes-bsp/rpi-u-boot-scr/files/boot.cmd.in

```diff
@@ -1,5 +1,31 @@
 fdt addr ${fdt_addr} && fdt get value bootargs /chosen bootargs
 fatload mmc 0:1 ${kernel_addr_r} @@KERNEL_IMAGETYPE@@
+
 if test ! -e @@BOOT_MEDIA@@ 0:1 uboot.env; then saveenv; fi;
-if env exists rpipart;then setenv bootargs "${bootargs} root=/dev/mmcblk0p${rpipart}"; fi
-@@KERNEL_BOOTCMD@@ ${kernel_addr_r} - ${fdt_addr}
+
+if env exists rpipart;
+then 
+  if test "${have_updated}" = 1; then
```

Comment:
nit: all of these `test` should use `-eq` for numeric comparison: e.g. if test $have_updated -eq 1; then